client/src/infographic/Infographic.js

```diff
@@ -188,6 +195,29 @@ class InfoGraph_ extends React.Component {
         include_tables: false,
       }}
     />;
+    const filter_icon_props = {
+      color: window.infobase_color_constants.primaryColor,
+      title: "Filter icon for panels",
+      width: 15,
+      height: 15,
+      vertical_align: 5,
+    };
+    const panel_renderers = !loading &&
+      _.chain(panel_keys)
+        .map((panel_key) => {
+          const panel_obj = PanelRegistry.lookup(panel_key, subject.level);
+          const panel_filter_arr = _.map(panel_filter, (value, key) => panel_filter[key] && key);
```

Comment:
`value === panel_filter[key]`? 

This also leaves a bunch of `false`s in the resulting array; unlikely to cause a bug but could if something causes `panel_obj.depends_on` to return something containing `false`. If you replace this with a chain than there's a few options to deal with that (this map followed by a `.compact()`, or even simpler just `_.chain(panel_filter).filter().keys().value()` I think).

Wasn't initially clear what `panel_filter_arr` is from the name, but it's something like `active_panel_filter_keys` (or `active_panel_data_dependencies`? since that's what filter keys are), right? If one of those is right, or you can think of an even clearer name, please update it.